### ISSUE TITLE
Support to drop table in personal database.

### ISSUE THREAD

**Issue by Taehui** [metatron-app/metatron-discovery#943]
**Is your feature request related to a problem? Please describe.**
- 개인데이터베이스(하이브)에서 저장된 테이블을 삭제하거나 이름을 수정하는 기능이 필요

**Describe the solution you'd like**
- 개인데이터베이스에 저장된 테이블을 삭제할 수 있는 기능 지원
- 개인데이터베이스에 저장된 테이블의 이름을 변경할 수 있는 기능 지원

**Comment by YooYoungmo** [comment]
#320 과 연결된 이슈입니다.
@kyungtaak @AnnieHwang 기획안 첨부 합니다. 피드백 부탁 드립니다.

---
![image](https://user-images.githubusercontent.com/16472109/49191583-6f34d480-f3ba-11e8-8e9a-5b13d3bdaf48.png)
---
![image](https://user-images.githubusercontent.com/16472109/49191595-76f47900-f3ba-11e8-9120-9093570b7f37.png)
---
![image](https://user-images.githubusercontent.com/16472109/49191600-7d82f080-f3ba-11e8-826f-4942d20a5225.png)
---
![image](https://user-images.githubusercontent.com/16472109/49191605-8378d180-f3ba-11e8-855a-04caf5665c41.png)

**Comment by i1befree** [comment]
이건 기능이 상당히 애매하네요. 원래 테이블 삭제 제어는 DB 에서 하는게 맞는게 아닌가 싶습니다. 워크벤치가 권한 제어를 할 수 있는 도구는 아니거든요. DDL 제어 기능을 워크벤치에 포함시키는 건 저도 필요하다고 봅니다만, 이렇게 특정 권한 제어를 넣는게 맞을지 모르겠습니다.

**Comment by Taehui** [comment]
@i1befree 맞는 말씀인데, 개인 데이터베이스를 지원하면서 발생할 수 밖에 없는 문제라고 생각합니다. 위 기획된 삭제가 개인 데이터베이스에만 해당하는 기능이라서요...보통 사용자가 워크벤치를 쓰는 패턴하고 다른 워크벤치와의 차별성이 생길려면, 개인 데이터베이스 지원은 꼭 필요하다고 생각하는데요.. 
검토하셔서 의견주세요~

**Comment by i1befree** [comment]
@Taehui 네. 말씀 하신 상황에선 어쩔 수 없을지 모르겠습니다만, 이런 기능을 넣는건 좀 곤란할 수 있어요. 특수한 기능이 들어가면 앞으로 계속 유지 보수를 해야 하는데, 이런 기능은 Tango 외에는 테스트 할 곳도 없게 되고, 오류가 계속 유지될 가능성이 무척 높지 않을까 걱정이되네요.
